compiler/test/dotc/tests.scala

```diff
@@ -153,6 +153,7 @@ class tests extends CompilerTest {
   @Test def pos_anonClassSubtyping = compileFile(posDir, "anonClassSubtyping", twice)
   @Test def pos_extmethods = compileFile(posDir, "extmethods", twice)
   @Test def pos_companions = compileFile(posDir, "companions", twice)
+  @Test def posVarargsT1625 = compileFiles(posDir + "varargsInMethodsT1625/")
```

Comment:
This is an unnecessary complication, just put your test sources in `tests/pos/` and remove this `@Test def ...`. Also use the prefix `i1625` for the test sources.